pkg/data/data.go

```diff
@@ -0,0 +1,94 @@
+package data
```

Comment:
It's good to include some documentation on what this package does. Similar to how it's done in other packages: https://github.com/EdwinWalela/africastalking-go/blob/0b1a0b87f1315f2b28f5764b745f424486509ded/pkg/sms/sms.go#L2